src/MostOpenLot.java

```diff
@@ -0,0 +1,19 @@
+
+public class MostOpenLot {
+
+	public static void main(String[] args) {
+		ParkingAPI parkingAPI = new ParkingAPI();
+		
+		
+		System.out.println(
+		parkingAPI.availableParkingLots(
+				parkingAPI.availableParkingLots(ParkingLot.WHITTNEY_LOT, ParkingLot.NORTH_LOT), 
+				ParkingLot.SOUTH_LOT));
+		
+		System.out.println(parkingAPI.predictedOpenSpots(
+				parkingAPI.predictedOpenSpots(ParkingLot.NORTH_LOT,ParkingLot.SOUTH_LOT), 
+				ParkingLot.WHITTNEY_LOT)
+				);
```

Comment:
This works, but there's no labels so when the program is run, it's hard for the user to know what it means. Could you add some additional print-lines that describe what's being printed?